classes/external.php

```diff
@@ -175,8 +175,8 @@ public static function get_notices_parameters() {
     }
 
     /**
-     * Gets a list of notices.                                                                      local/sitenotice/tests/sitenotice_test.php
-
+     * Gets a list of notices.
+     * local/sitenotice/tests/sitenotice_test.php
```

Comment:
If you're cleaning stuff up, then this probably should be removed. Does look like someone accidentally pasted it here.

Reply:
Thanks for catching that @dmitriim !